src/status_im/infra/market/transform.cljs

```diff
@@ -0,0 +1,70 @@
+(ns status-im.infra.market.transform
```

Comment:
Here we have all transformations of market data between status-go backend to to mobile structures.
Malli schemas can be added later to ensure API is not broken